### ISSUE TITLE
Meter Example: Document and test pause button

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-practices#1222]
We added a pause button to the meter example. It is not documented under keyboard interaction. It is not tested. Perhaps we should also add an accessibility features section that describes the rationale for including it.

**Comment by ZoeBijl** [comment]
The pause button doesn’t need to be included in the keyboard interaction I don’t think. It’s not part of the pattern, only the example itself. The [`alert` code example](https://w3c.github.io/aria-practices/examples/alert/alert.html) has a button to trigger the alert. But it doesn’t have that detailed in the keyboard interaction. It does have an accessibility features section and I’m in favour of adding that here too.

Undecided about the `aria-hidden`. It’s also not part of the pattern but rather the code example. Perhaps that could be included in the accessibility features section too?

**Comment by a11ydoer** [comment]
Here is the completed [meter](https://htmlpreview.github.io/?https://github.com/w3c/aria-practices/blob/master/examples/meter/meter.html)
To my understanding, documentation and coding example works are done. 
I think I was supposed to do regression tesing... @spectranaut can confirm whether we need the regression testing for this example?
